e2e/tests/nightly/gke-apps/cromwell.spec.ts

```diff
@@ -23,13 +23,20 @@ const cromshellSubmitRCmd = (cromshell_version) =>
   `system2('${cromshell_version}', args = c('submit', ` +
   `'${wdlFileName}','${jsonFileName}'), stdout = TRUE, stderr = TRUE)`;
 
+const cromshellStatusPythonCmd = (cromshell_version, cromshellJobId) =>
+  `!${cromshell_version} status ${cromshellJobId}`;
+const cromshellStatusRCmd = (cromshell_version, cromshellJobId) =>
+  `system2('${cromshell_version}', args = c('status', '${cromshellJobId}'), stdout = TRUE, stderr = TRUE)`;
+
+const workspaceName = 'e2eCromwellTest';
+
 describe('Cromwell GKE App', () => {
   beforeEach(async () => {
     await signInWithAccessToken(page);
   });
 
   test('Create and delete a Cromwell GKE app', async () => {
-    await findOrCreateWorkspace(page, { workspaceName: 'e2eCreateCromwellGkeAppsPanelTest' });
+    await findOrCreateWorkspace(page, { workspaceName: workspaceName });
```

Comment:
```suggestion
    await findOrCreateWorkspace(page, { workspaceName });
```

Reply:
What do you think about moving this into the `beforeEach` ?

Reply:
Done